Add files via upload

yueyingn opened the pull request:
Compare GSMF from current observation, need package astrodatapy installed.

yueyingn committed:
Add files via upload

compare GSMF from current observation, need astrodatapy installed.

sbird commented:
Thanks! Can you please copy-paste this to a plain python (not notebook) form? It's easier on the version control system.

yueyingn committed:
Add files via upload

The script to plot GSMF and compare with current observation. Need astrodatapy pre-installed.

yueyingn commented:
Sure, here it is.

sbird commented:
Merged with some edits!
